src/czechpmdevs/multiworld/generator/void/VoidGenerator.php

```diff
@@ -38,7 +38,7 @@ public function generateChunk(ChunkManager $world, int $chunkX, int $chunkZ): vo
 		$chunk = $world->getChunk($chunkX, $chunkZ);
 
 		if($chunkX == 16 && $chunkZ == 16) {
-			$chunk->setFullBlock(0, 64, 0, BlockLegacyIds::GRASS << 4);
+			$chunk->setFullBlock(0, 64, 0, BlockTypeIds::GRASS << 4);
```

Comment:
![image](https://user-images.githubusercontent.com/40556088/178101990-904a40f0-92e2-4410-ae10-5929a8d599d3.png)

There should be grass block instead of update one

Reply:
Fixed